sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -192,6 +189,8 @@
 					fluidRowLocs(CaseDataDto.FOLLOW_UP_COMMENT);
 
 	private static final String COMMENTS_HTML_LAYOUT =
+			loc(PAPER_FORM_DATES_LOC) +
```

Comment:
This is semantically wrong; the paper form fields shouldn't be in something that's called "COMMENTS_HTML_LAYOUT". Please either find a new name for this field or split it into two fields (COMMENTS_LAYOUT and PAPER_FORM_DATES_LAYOUT).

Reply:
Yes, you're right.